x/derivatives/module.go

```diff
@@ -123,7 +127,7 @@ func (am AppModule) LegacyQuerierHandler(_ *codec.LegacyAmino) sdk.Querier {
 
 // RegisterServices registers a gRPC query service to respond to the module-specific gRPC queries
 func (am AppModule) RegisterServices(cfg module.Configurator) {
-	types.RegisterMsgServer(cfg.MsgServer(), keeper.NewMsgServerImpl(am.keeper))
+	// types.RegisterMsgServer(cfg.MsgServer(), keeper.NewMsgServerImpl(am.keeper))
```

Comment:
Why is this commented out?

Reply:
didn't dig in deeply, 
but I assume, the design of the msg services in cosmos sdk (and tendermint?) had changed significantly at v0.46 and it affected the way to register msgs.

I can search deeply if you need.